ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/SlidingWindowRateLimiter.java

```diff
@@ -74,6 +87,27 @@ public SlidingWindowRateLimiter(final int requestLimitInMB, final long slidingWi
     this.slidingWindowSizeMs = slidingWindowSizeMs;
     this.responseSizesLog = new LinkedList<>();
     this.numBytesInWindow = 0;
+
+    metrics.addMetric(
+        new MetricName(
+            metricNamespace + "-bandwidth-limit-remaining",
+            ReservedInternalTopics.KSQL_INTERNAL_TOPIC_PREFIX + "limits",
+            "The current value of the bandwidth limiter",
+            metricsTags
+        ),
+        (metricConfig, l) -> throttleLimit - numBytesInWindow
```

Comment:
Is this a guage that will just report the current bytes remaining?

Reply:
Yep!